src/number.rs

```diff
@@ -1,7 +1,47 @@
 // Exports of <CoreFoundation/CFNumber.h>
 
+use libc::c_void;
+
 use base::*;
 
+#[repr(C)]
+struct __CFBoolean;
+pub type CFBooleanRef = *const __CFBoolean;
+
+extern "C" {
+    pub fn CFBooleanGetTypeID() -> CFTypeID;
+    pub fn CFBooleanGetValue(boolean: CFBooleanRef) -> Boolean;
+}
+
+pub type CFNumberType = CFIndex;
+pub const kCFNumberSInt8Type: CFNumberType = 1;
+pub const kCFNumberSInt16Type: CFNumberType = 2;
+pub const kCFNumberSInt32Type: CFNumberType = 3;
+pub const kCFNumberSInt64Type: CFNumberType = 4;
+pub const kCFNumberFloat32Type: CFNumberType = 5;
+pub const kCFNumberFloat64Type: CFNumberType = 6;
+pub const kCFNumberCharType: CFNumberType = 7;
+pub const kCFNumberShortType: CFNumberType = 8;
+pub const kCFNumberIntType: CFNumberType = 9;
+pub const kCFNumberLongType: CFNumberType = 10;
+pub const kCFNumberLongLongType: CFNumberType = 11;
+pub const kCFNumberFloatType: CFNumberType = 12;
+pub const kCFNumberDoubleType: CFNumberType = 13;
+pub const kCFNumberCFIndexType: CFNumberType = 14;
+pub const kCFNumberNSIntegerType: CFNumberType = 15;
+pub const kCFNumberCGFloatType: CFNumberType = 16;
+pub const kCFNumberMaxType: CFNumberType = 16;
+
+#[repr(C)]
+struct __CFNumber;
+pub type CFNumberRef = *const __CFNumber;
+
 extern "C" {
     pub fn CFNumberGetTypeID() -> CFTypeID;
+    pub fn CFNumberCreate(alloc: CFAllocatorRef, theType: CFNumberType, valuePtr: *mut c_void) -> CFNumberRef;
```

Comment:
Oops. Looks like this got changed from `*const` to `*mut` in your last commit. Should be `*const`.